GNUmakefile

```diff
@@ -148,6 +138,15 @@ goreleaser-dev-build: ## Run goreleaser snapshot build
 goreleaser-dev-release: ## run goreleaser snapshot release
 	./tools/bin/goreleaser_wrapper release --snapshot --rm-dist --config ${GORELEASER_CONFIG}
 
+.PHONY: clean
+clean: ## Remove build files
```

Comment:
Not sure if this is needed, you can use `git clean` instead and it won't have to be updated.